airbyte-proxy/nginx-no-auth.conf.template

```diff
@@ -24,4 +24,16 @@ http {
       proxy_pass "${PROXY_PASS_API}";
     }
   }
+
+    server {
+      listen 8003;
+
+      location / {
+        proxy_set_header Host $host;
+        proxy_set_header X-Real-IP $remote_addr;
+        proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
+
+        proxy_pass "${CONNECTOR_BUILDER_SERVER_API}";
```

Comment:
forward to "http://airbyte-connector-builder-server:80" as defined in https://github.com/airbytehq/airbyte/pull/19264/files#diff-8a4136717271f90d70120fef7bb8904d26f3fbcfe7520dbd39fa0b8723d8548eR15